app/main.py

```diff
@@ -195,7 +193,7 @@ def setEdge(dataDump):
         dataDump (list): Converted JSON data
 
     Returns:
-         Gameboard with the edges initialised to '2'
+        Gameboard with the edges initialised to '2'
```

Comment:
This has been changed to 10. Not relevant to this pull request but should be known for future documentation

Reply:
Should be changed to refer to the variable not the number. I did notice it but I didn't change it here because it was out of the scope of this ticket.